### PR TITLE
Polish the Chinese documentation of trusted committer

### PR DESCRIPTION
**Pull request by WillemJiang**
Fixed the Contributor link format issues
Polished some translation of Chinese according to the English documentation.

### CODE REVIEW

**Review by rrrutledge — APPROVED** [review]
Hi @WillemJiang, thank for this help!  Things look good from my reading!  Although I am not a native Chinese speaker, I checked out some of the formatting changes that you made.  I left a few suggestions for improvement also.

A native Chinese speaker should review as well.  I will tag a few that I know on this PR.

Thank you!!

**Comment by rrrutledge** [comment]
@adadai88, @niminghong, @yvonneT1 or @tanzhongyi003 may be interested in this review.

File: trusted-committer/zh/02-ensuring-product-quality-zh.asciidoc [hunk 1]
```diff
@@ -4,17 +4,17 @@
 
 在InnerSource社区中，Trusted Committer拥有所有与技术有关的决策权，尤其是与产品质量有关的决策权。这意味着他们需要确保社区内成员在遵循这些决策。这包括在社区内部和外部进行交流，并在必要时公示这些决策。但是，Trusted Committer并不一定要自己做出所有与技术相关的决定，也不一定要负责所有的执行工作。
 
-Trusted Committer的工作是在其社区中交流和阐明质量标准，并以https://innersourcecommons.org/resources/learningpath/contributor/index[_贡献者 Contributor_]可理解和可操作的方式制定这些标准。这其中当然会包括书写文档，但是我们认为可信提交者传达这些质量标准的最有效方法是举例子。我们认为，InnerSource社区尝试将自己与传统软件开发项目区分开来，不仅在于他们组织开发的方式，而且还包括他们生产的软件的质量，这是一个非常有价值的目标。高水平的软件质量对于在InnerSource社区中建立和维持部分用户及其管理层的信任至关重要。我们都知道低质量的发布是会打击人们对社区的信任感的。
+Trusted Committer的工作是在其社区中交流和阐明质量标准，并以 https://innersourcecommons.org/resources/learningpath/contributor/index[_贡献者（Contributor）_]可理解和可操作的方式制定这些标准。这其中当然会包括书写文档，但是我们认为可信提交者传达这些质量标准的最有效方法是举例子。我们认为，InnerSource社区尝试将自己与传统软件开发项目区分开来，不仅在于他们组织开发的方式，而且还包括他们生产的软件的质量，这是一个非常有价值的目标。高水平的软件质量对于在InnerSource社区中建立和维持部分用户及其管理层的信任至关重要。我们都知道低质量的发布是会打击人们对社区的信任感的。
```
[the remark this answers is not in the document]
Comment (WillemJiang) [reply trusted-committer/zh/02-ensuring-product-quality-zh.asciidoc]: Done,  I also updated other links which have the Chinese version.

File: trusted-committer/zh/03-keeping-the-community-healthy-zh.asciidoc [hunk 2]
```diff
@@ -2,21 +2,21 @@
...
+健康的社区是什么样的？很简单，在一个健康的社区中， https://innersourcecommons.org/resources/learningpath/contributor/index[_贡献者（Contributor）_]往往会持续进行贡献，将大部分时间花费在开发软件上，并从中提升自己的能力。这样的话，一个健康的社区将能不断发展。
 
-为什么贡献者（contributor）会加入并停留在社区中？有些人这样做是因为他们认同社区的宗旨或使命。清楚地表达和贯彻社区的宗旨是Trusted Committe的工作。人们通常认识不到这一点的重要性，但在营销社区及其产品的过程中，这确实是必不可少的。
+为什么贡献者（Contributor）会加入并停留在社区中？有些人这样做是因为他们认同社区的宗旨或使命。清楚地表达和贯彻社区的宗旨是Trusted Committe的工作。人们通常认识不到这一点的重要性，但在营销社区及其产品的过程中，这确实是必不可少的。
 
-人们留下来的另一个更关键的原因是，他们喜欢与社区中的其他成员（包括Trusted Committe）一起工作。在一个繁荣的社区里成员间通常会相互尊重和相互沟通。贡献被视为馈赠或付出，而不是娱乐，而出色的（尤其是NO.1）贡献者（contributor）通常会受到称赞。Trusted Committe在所有这方面的工作主要是为他人树立榜样，就是为预期的软件质量水平树立榜样。如有必要，Trusted Committe可以为社区制定并制定行为准则。如果有某些社区成员的行为对社区健康有所损害，那么Trusted Committe有责任去解决此​​问题。Trusted Committe应为人们创造机会，使他们能够定期（线上或线下）聚在一起，彼此了解并在有冲突时和平解决。
+人们留下来的另一个更关键的原因是，他们喜欢与社区中的其他成员（包括Trusted Committe）一起工作。在一个繁荣的社区里成员间通常会相互尊重和相互沟通。贡献被视为馈赠或付出，而不是娱乐，而出色的（尤其是NO.1）贡献者（contributor）通常会受到称赞。Trusted Committe在所有这方面的工作主要是为他人树立榜样，就是为预期的软件质量水平树立榜样。如有必要，Trusted Committe可以为社区制定并执行行为准则。如果有某些社区成员的行为对社区健康有所损害，那么Trusted Committe有责任去解决此类问题。Trusted Committe应为人们创造机会，使他们能够定期（线上或线下）聚在一起，彼此了解并在有冲突时和平解决。
```
[the remark this answers is not in the document]
Comment (WillemJiang) [reply trusted-committer/zh/03-keeping-the-community-healthy-zh.asciidoc]: Fixed the typos as suggestion.

[the remark this answers is not in the document]
Comment (rrrutledge) [reply trusted-committer/zh/03-keeping-the-community-healthy-zh.asciidoc]: Great!

**Review by rrrutledge — APPROVED** [review]
Great!  We can see if one of the native Chinese speakers is online today to review.

**Review by tanzhongyi003 — APPROVED** [review]
LGTM

**Comment by lenucksi** [comment]
Excellent, thank your for the updates and polishing @WillemJiang and thanks for the LGTM @tanzhongyi003 :tada:

**Pull request merged**